processing/src/main/java/org/apache/druid/collections/BlockingPool.java

```diff
@@ -49,4 +49,9 @@ public interface BlockingPool<T>
    * @return count of pending requests
    */
   long getPendingRequests();
+
+  /**
+   * @return number of used buffers from the pool
+   */
+  long getUsedBufferCount();
```

Comment:
BlockingPool is generic. It shouldn't reference buffers. 
```suggestion
  long getUsedResourcesCount();
```